.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: php
+
+sudo: false
+
+cache:
+  directories:
+    - "$HOME/.composer/cache"
+
+env:
+  - COMPOSER_FLAGS="--prefer-stable"
+
+matrix:
+  include:
+    - php: 5.3
+      env: COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
--prefer-lowest needs --prefer-stable also, so you don't test with alpha/beta versions
imo, the best practice is to combine -prefer-lowest with the highest PHP version, so that you're sure that your lowest deps still work with the latest PHP